apis/swagger.yml

```diff
@@ -266,13 +271,20 @@ paths:
         - name: id
           in: path
           required: true
-          description: "ID of piece"
+          description: "ID of task"
           type: string
+        - name: num
```

Comment:
The field `num` means that the pieceNum of task, in the meantime, we use `num` as the request number of pieces of task in API ` /tasks/{id}/pieces`, shall we make a difference?

Reply:
What is the num format like? Could you give us an example? @lowzj

Reply:
We can use `pieceRange` here to identify a piece, and the type is string , an example: `0-45565`.

Reply:
`pieceRange` is better. `Range` is a http header, for example: `Range: bytes=0-45565`.